HIP-xxxx.md

```diff
@@ -0,0 +1,41 @@
+# HIP-xxxx : Name data standard for hsd resolver plugins
+
+```
+Number:  HIP-xxxx
+Title:   Name data standard for hsd resolver plugins
+Type:    Informational
+Status:  Draft
+Authors: Mike Carson <https://impervious.com/>
+Created: 2021-02-08
+```
+
+## Abstract
+
+This HIP describes how to set the data for a Handshake name so that it resolves with the help of an [hsd](https://github.com/handshake-org/hsd) plugin.
+
+## Motivation
+
+There may be various reasons to resolve a name with the help of an hsd plugin.  One example is for decentralized subdomains on Ethereum.  An hsd plugin for this would recognize the data on a name in a way to resolve it with Ethereum.
+
+## Name data standard
+
+We propose setting the NS records for the name with an underscore suffix which will resolve to a specific plugin.  The prefix will be used as data for the plugin. For example:
```

Comment:
I see how this works for eth but I'm curious how this prefix dot suffix format was chosen, and what considerations were made. since it could be conflated with a DNS name I thought it might plausibly cause some potential problems with other systems that aren't aware of HSD  plugins but still end up processing the data in a normal DNS context. normally non-dns stuff is done via TXT records in a key=value or json format

Reply:
I can assume the format was chosen to be `<data>.<invalid domain>`.

`_eth` is not a valid TLD and will never be accepted as a Handshake name, but can be used to trigger the plugin.